public/locales/en/modules/torrents-status.json

```diff
@@ -39,12 +39,12 @@
     },
     "errors": {
       "noDownloadClients": {
-        "title": "No supported download clients found!",
-        "text": "Add a download service to view your current downloads"
+        "title": "No supported Torrent Download clients found!",
```

Comment:
Same as above it's a torrent client

Reply:
I based this of how it's named in the docs, docs will have to be changed.

Reply:
Changed in docs too.